tests/endpoint-test.js

```diff
@@ -1293,7 +1293,69 @@ describe('Endpoint', function() {
 				});
 			});
 		});
+	});
+
+
+	describe('Collaboration Whitelists', function() {
+		describe('create()', function() {
+			it('should make a post request to create a domain collaboration whitelisting', function() {
+				var post_fixture = getFixture('collaboration-whitelists/post_collaboration_whitelists_200'),
```

Comment:
The upcoming linter changes are going to complain about snake_case variable names — these should be camelCase